src/Mod/TechDraw/TechDrawTools/TaskCopyView.py

```diff
@@ -0,0 +1,141 @@
+# *                                                                         *
+# *   Copyright (c) 2022 - Wanderer Fan <wandererfan@gmail.com>             *
+# *                                                                         *
+# *   This program is free software; you can redistribute it and/or modify  *
+# *   it under the terms of the GNU Lesser General Public License (LGPL)    *
+# *   as published by the Free Software Foundation; either version 2 of     *
+# *   the License, or (at your option) any later version.                   *
+# *   for detail see the LICENCE text file.                                 *
+# *                                                                         *
+# *   This program is distributed in the hope that it will be useful,       *
+# *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+# *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+# *   GNU Library General Public License for more details.                  *
+# *                                                                         *
+# *   You should have received a copy of the GNU Library General Public     *
+# *   License along with this program; if not, write to the Free Software   *
+# *   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  *
+# *   USA                                                                   *
+# *                                                                         *
+# ***************************************************************************
+"""Provides the TechDraw CopyView Task Dialog."""
+
+__title__ = "TechDrawTools.TaskCopyView"
+__author__ = "WandeererFan"
+__url__ = "http://www.freecadweb.org/index.html"
+__version__ = "00.01"
+__date__ = "2022/01/11"
+
+from PySide.QtCore import QT_TRANSLATE_NOOP
+from PySide import QtCore
+import PySide.QtGui as QtGui
+
+import FreeCAD as App
+import FreeCADGui as Gui
+import TechDraw
+from TechDrawTools import TDToolsUtil
+from TechDrawTools import TDToolsMovers
+import os
+
+class TaskCopyView:
+    def __init__(self):
+        import os
+        self._uiPath = App.getHomePath()
+        self._uiPath = os.path.join(self._uiPath, "Mod/TechDraw/Gui/TaskMoveView.ui")
+        self.form = Gui.PySideUic.loadUi(self._uiPath)
+
+        self.form.setWindowTitle(QT_TRANSLATE_NOOP("CopyView", "Copy View to a second Page"))
+
+        self.form.pbView.clicked.connect(self.pickView)
+        self.form.pbFromPage.clicked.connect(self.pickFromPage)
+        self.form.pbToPage.clicked.connect(self.pickToPage)
+
+        self.viewName = ""
+        self.fromPageName = ""
+        self.toPageName   = ""
+
+    def accept(self):
+#        print ("Accept")
+        v = App.ActiveDocument.getObject(self.viewName)
+        f = App.ActiveDocument.getObject(self.fromPageName)
+        t = App.ActiveDocument.getObject(self.toPageName)
+        TDToolsMovers.moveView(v, f, t, True)
```

Comment:
Previously you called the variables other names. Consider reusing the name for readability
```suggestion
        vName = App.ActiveDocument.getObject(self.viewName)
        fpName = App.ActiveDocument.getObject(self.fromPageName)
        tpName = App.ActiveDocument.getObject(self.toPageName)
        TDToolsMovers.moveView(vName, fpName, tpName, True)
```